app/controllers/deploys_controller.rb

```diff
@@ -125,4 +128,8 @@ def find_project
   def find_deploy
     @deploy = Deploy.find(params[:id])
   end
+
+  def active_deploy_scope
```

Comment:
might be more reusable with `def deploy_scope; @project.try(:deploys) || Deploy` and then `deploy_scope.active` but either way is fine